<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-570PAFEV" failures="0" tests="1" name="Test" time="0.136" errors="0" timestamp="2023-06-19T23:43:26 IST">
  <testcase classname="org.capstonetestcase.TestCaseOne" name="@BeforeClass initialSetup" time="0.085">
    <failure type="java.net.MalformedURLException" message="Error at index 4 in: &amp;quot;4444.&amp;quot;">
      <![CDATA[java.net.MalformedURLException: Error at index 4 in: "4444."
at java.base/java.net.URL.<init>(URL.java:706)
at java.base/java.net.URL.<init>(URL.java:568)
at java.base/java.net.URL.<init>(URL.java:515)
at Hritik.Capstone_Automation.BaseconfigCapstone.setUp(BaseconfigCapstone.java:29)
at org.capstonetestcase.TestCaseOne.initialSetup(TestCaseOne.java:34)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
Caused by: java.lang.NumberFormatException: Error at index 4 in: "4444."
at java.base/java.lang.NumberFormatException.forCharSequence(NumberFormatException.java:84)
at java.base/java.lang.Integer.parseInt(Integer.java:735)
at java.base/java.net.URLStreamHandler.parseURL(URLStreamHandler.java:232)
at java.base/java.net.URL.<init>(URL.java:701)
... 25 more
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeClass initialSetup -->
  <testcase classname="org.capstonetestcase.TestCaseOne" name="tc1Scenarios" time="0.0">
    <skipped/>
  </testcase> <!-- tc1Scenarios -->
</testsuite> <!-- Test -->
